README.md

```diff
@@ -1,2 +1,35 @@
-# nlp-in-10-lines
-Repository for PyCons 2016 workshop Natural Language Processing in 10 Lines of Code.
+### PyCon UK 2016 
+![NLP in 10 Lines of Code](http://i.imgur.com/TuusTAm.png)
```

Comment:
I would host this image on Github and add relative link to image hosted in this repo.